test/prepare-updates-img

```diff
@@ -14,6 +14,9 @@ for i in ${TEST_RPMS//,/ }; do
     cp $i tmp/rpms
 done
 
+# Copy the payload RPM to the tmp/rpms
+cp "$(readlink -f bots/images/fedora-rawhide-anaconda-payload)" tmp/rpms/
```

Comment:
You can also do: `cp -L bots/images/fedora-rawhide-anaconda-payload tmp/rpms/fedora-rawhide-anaconda-payload.rpm`

Feel free top decide, I think the new suggestion is cleaner.